server/migrations/20180105175431_add_session_table.js

```diff
@@ -0,0 +1,13 @@
+exports.up = knex => (
+  knex.raw(`
+    CREATE TABLE "session" (
```

Comment:
Have you considered JWT token sessions? They'd be more lightweight since no database is involved in the session management

Reply:
Not at all. Is there a good library for them with express-session which would also work with `socket.io`? Is there any other benefit than no database.